rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -873,13 +928,28 @@ error_code sys_fs_stat(ppu_thread& ppu, vm::cptr<char> path, vm::ptr<CellFsStat>
 		}
 	}
 
-	sb->mode = info.is_directory ? CELL_FS_S_IFDIR | 0777 : CELL_FS_S_IFREG | 0666;
+	sb->mode = item_count != 1 || mp == &g_mp_sys_app_home ? info.is_directory ? CELL_FS_S_IFDIR | 0777 : CELL_FS_S_IFREG | 0666 : CELL_FS_S_IFDIR | 0755;
 	sb->uid = mp->flags & lv2_mp_flag::no_uid_gid ? -1 : 0;
 	sb->gid = mp->flags & lv2_mp_flag::no_uid_gid ? -1 : 0;
-	sb->atime = info.atime;
-	sb->mtime = info.mtime;
-	sb->ctime = info.ctime;
-	sb->size = info.size;
+
+	if (mp != &g_mp_sys_dev_bdvd)
+	{
+		sb->atime = info.atime;
+		sb->mtime = info.mtime;
+		sb->ctime = item_count == 1 && mp != &g_mp_sys_app_home ? 0 : info.ctime;
+	}
+	else
+	{
+		fs::stat_t bdvd_info{};
+		verify(HERE), fs::stat(vfs::get("/dev_bdvd"sv), bdvd_info);
```

Comment:
Umm... what? Where is this coming from? It doesn't make any sense since blu ray do have mtime information for files.

Reply:
Maybe it should store mtime to atime and ctime for each, since its read-only it shouldn't change while opening.

Reply:
I removed those changes, the ones remianed from sys_fs are in sys_fs_stat and they are based on hw tests which is why the specific value flags.